### PR TITLE
Grammatically updated the tech docs

### PR DESCRIPTION
**Pull request by Clarksoncdc**
Hello,

I am currently working on my cyber security BA, and this class project was to find and contribute to a open source project. I wanted to keep it simple the first go around to get the process down, and the changes that I made are strictly grammatical. What I wanted to accomplish was to effectively contribute to this project in the from of readability in the tech documentation. I made changes to the tech docs because I felt the wording could be a little more clean and precise as to what actions are being performed.

The changes made were strictly from a personnel perspective. In the future I would like to gain more knowledge and skill in order to work towards issue tickets for Borg. Thank you for your time, it is much appreciated. 

Links below for screenshots;

 - git diff
 - image
 - NEWNEWComparision

### CODE REVIEW

**Review by fantasya-pbem — COMMENTED** [review]
I do not think your changes make it better. The wording may be more professional, but the semantics is less precise.

File: docs/quickstart_example.rst.inc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-1. Before a backup can be made a repository has to be initialized::
+1. Before a backup can be converted into a repository it needs to be initialized (init)::
```
[the remark this answers is not in the document]
Comment (ThomasWaldmann) [reply docs/quickstart_example.rst.inc:1]: @Clarksoncdc i don't think you've understood the phrase as it was meant. Is there maybe an important comma missing in the phrase, should it be:

```
Before a backup can be made, a repository has to be initialized::
```

File: docs/quickstart_example.rst.inc [hunk 2]
```diff
@@ -11,9 +11,9 @@
 
     $ borg create --stats /path/to/repo::Tuesday ~/src ~/Documents
 
-   This backup will be a lot quicker and a lot smaller since only new never
-   before seen data is stored. The ``--stats`` option causes Borg to
-   output statistics about the newly created archive such as the amount of unique
+   This backup will increase efficiency as the load is smaller and faster while
```
[the remark this answers is not in the document]
Comment (ThomasWaldmann) [reply docs/quickstart_example.rst.inc:14]: I don't see an improvement here, the proposed change describes the behaviour less precise than the original text.

So, if there is a grammar issue in the original text, just fix that?

**Comment by ThomasWaldmann** [comment]
@Clarksoncdc did you see the feedback?

**Comment by ThomasWaldmann** [comment]
Ping.

**Comment by ThomasWaldmann** [comment]
Guess this is going nowhere...

**Pull request closed without merging**